man/mlpack-serialization.Rd

```diff
@@ -0,0 +1,22 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/serialization.R
+\name{Serialize}
```

Comment:
I can't respond to https://github.com/Yashwants19/RcppMLPACK/pull/8#discussion_r443146708 while writing this review but I'd place a vote for `mlpack_serialize()`; however, my vote should not count for much as I'm not familiar with what R conventions are.  Really our goal should be to make something that feels as "R-native" as possible, and so if `Serialize()` gets us closer than `mlpack_serialize()`, I suppose that's the better option.

Reply:
There isn't so much of a convention outside of the uppercase letter. As an example, here's one based on `keras`:

https://tensorflow.rstudio.com/reference/keras/serialize_model/

Long story short, we can revisit this during the touchup period prior to submission to _CRAN_. The important functionality of saving/exporting is present.